XMonad/Hooks/StatusBar.hs

```diff
@@ -377,7 +377,7 @@ statusBarPipe cmd xpp = do
 --
 -- > xmobarrc_top
 -- > Config { ...
--- >        , commands = [ Run XPropertyLog "_XMONAD_LOG_1", ... ]
+-- >        , commands = [ Run (XPropertyLog "_XMONAD_LOG_1"), ... ]
```

Comment:
I don't think that this is needed when one just uses a config file for xmobar (e.g., see [here](https://github.com/liskin/dotfiles/blob/home/.xmobarrc#L17)), which is what the documentation is concerned with

Reply:
I think so too, thanks for the contribution @chezbgone but I'd keep it as is

Reply:
I see, I actually use a "pure Haskell config" (described in 9.3 of https://xmobar.org/#configurations-written-in-pure-haskell), which gave me a type error for the example given, so I was a bit confused.

I agree that we should keep the original as slotThe noted, since the documentation should be for standard configurations, and also the type error was quite easy to resolve.